packages/mui-material/src/Rating/Rating.js

```diff
@@ -326,7 +326,8 @@ const defaultIcon = <Star fontSize="inherit" />;
 const defaultEmptyIcon = <StarBorder fontSize="inherit" />;
 
 function defaultLabelText(value) {
-  return `${value} Star${value !== 1 ? 's' : ''}`;
+  const _value = value || 0;
+  return `${_value} Star${_value > 1 ? 's' : ''}`;
```

Comment:
This would give us:
- ❌ `null` (or any falsy value) => `"0 Star"` (it should be in plural)
- ❌ `0` => `"0 Star"` (it should be in plural)
- ✅ `1` => `"1 Star"`
- ✅ `2` => `"2 Stars"`


If we change it to this, we get better results:

```suggestion
  return `${value || "0"} Star${value !== 1 ? 's' : ''}`;
```

- ✅ `null` (or any falsy value) => `"0 Stars"`
- ✅ `0` => `"0 Stars"`
- ✅ `1` => `"1 Star"`
- ✅ `2` => `"2 Stars"`

Reply:
Thanks @aarongarciah 

LGTM, updated😎